discovery/consul/consul.go

```diff
@@ -43,6 +43,8 @@ const (
 	nodeLabel = model.MetaLabelPrefix + "consul_node"
 	// metaDataLabel is the prefix for the labels mapping to a target's metadata.
 	metaDataLabel = model.MetaLabelPrefix + "consul_metadata_"
+	// serviceMetaDataLabel is the prefix for the labels mapping to a target's service metadata.
+	serviceMetaDataLabel = model.MetaLabelPrefix + "consul_service_metadata_"
```

Comment:
Don't forget to add that to the documentation

Reply:
Done.